src/index.html

```diff
@@ -11,12 +11,104 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="lists">
+          <li class="is-active list first-list">
+            <a
+              href="#"
+              class="link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="is-active list">
+            <a
+              href="#"
+              class="link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="is-active list">
+            <a
+              href="#"
+              class="link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="is-active list center-list">
+            <a
+              href="#"
+              class="link"
+              data-qa="hover"
+              id="only-color"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="is-active list">
+            <a
+              href="#"
+              class="link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="is-active list">
+            <a
+              href="#"
+              class="link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="is-active list">
+            <a
+              href="#"
+              class="link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="is-active list last-list">
+            <a
+              href="#"
+              class="link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
There should be empty lines between sibling elements for readability. However, don't add empty lines between parent and child elements.